packages/bruno-app/src/components/CollectionSettings/index.js

```diff
@@ -1,21 +1,21 @@
-import React from 'react';
```

Comment:
Default sort ordering in VS Code, will revert if there is a special code style rule for it in the project.